tests/core/test_policies.py

```diff
@@ -663,6 +664,89 @@ def test_augmented_prediction(
             model_storage=model_storage,
             resource=resource,
             execution_context=execution_context,
+            config={"max_history": max_history},
```

Comment:
Here and elsewhere we could use the `POLICY_MAX_HISTORY` constant instead of the `max_history` string.